.github/workflows/label-conflicting-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: "Label conflicting PRs"
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+    
+permissions:
+  pull-requests: read
```

Comment:
Did you test this? Adding labels seems to be a write operation, see also https://github.com/eps1lon/actions-label-merge-conflict/issues/81 and https://github.com/eps1lon/actions-label-merge-conflict#continueonmissingpermissions

Reply:
> test it again on my dummy repo

Be sure to try filing a PR from a fork as a user who lacks write permission to the repo.

Reply:
> > test it again on my dummy repo
> 
> Be sure to try filing a PR from a fork as a user who lacks write permission to the repo.

Testing with write permissions for PRs works fine so far on my account: https://github.com/NotMyFault-org/jenkins-conflicts-action-test/pull/13

However, I needed to add the boolean option Daniel mentioned in order to get it working for users without write permissions to the repository: https://github.com/NotMyFault-org/jenkins-conflicts-action-test/pull/15

If `continueOnMissingPermissions` suffices, can we set the permissions to read only for PRs again to wipe write access for all other scopes 🤔